modules/swagger-codegen/src/main/resources/Java/ApiClient.mustache

```diff
@@ -66,7 +66,7 @@ public class ApiClient {
     objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
     objectMapper.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
     objectMapper.enable(DeserializationFeature.READ_ENUMS_USING_TO_STRING);{{#java8}}
-    objectMapper.registerModule(new JavaTimeModule());{{/java8}}{{^java8}}
+    objectMapper.registerModule(new JSR310Module());{{/java8}}{{^java8}}
```

Comment:
The generated pom used a slightly older version of the jackson JSR310 library - which didn't have the newer JavaTimeModule.  However, I take your point and have updated the dependency and the code in my latest PR.